azext_iot/_help.py

```diff
@@ -912,3 +912,253 @@
     type: command
     short-summary: Delete a device registration in an Azure IoT Hub Device Provisioning Service.
 """
+
+helps['iot dt'] = """
+    type: group
+    short-summary: Manage digital twin of an IoT Plug and Play device.
+"""
+
+helps['iot dt invoke-command'] = """
+    type: command
+    short-summary: Executes a command on an IoT Plug and Play device.
+    long-summary: You can leverage az login and provide --hub-name instead of --login for every command.
+    examples:
+    - name: Execute a command on device .
+      text: >
+        az iot dt invoke-command --login {iothub_cs}
+        --interface {plug_and_play_interface} --device-id {device_id}
+        --command-name {command_name} --command-payload {payload}
+    - name: Execute a command on device within current session.
+      text: >
+        az iot dt invoke-command --hub-name {iothub_name}
+        --interface {plug_and_play_interface} --device-id {device_id}
+        --command-name {command_name} --command-payload {payload}
+"""
+
+helps['iot dt list-interfaces'] = """
+    type: command
+    short-summary: List interfaces of a target IoT Plug and Play device.
+    long-summary: You can leverage az login and provide --hub-name instead of --login for every command.
+    examples:
+    - name: List all IoT Plug and Play interfaces on a device.
+      text: >
+        az iot dt list-interfaces --login {iothub_cs}
+        --device-id {device_id}
+    - name: List all IoT Plug and Play interfaces on a device within current session.
+      text: >
+        az iot dt list-interfaces --hub-name {iothub_name} --device-id {device_id}
+"""
+
+helps['iot dt list-properties'] = """
+    type: command
+    short-summary: List properties of a target IoT Plug and Play device interface(s).
+    long-summary: You can leverage az login and provide --hub-name instead of --login for every command.
+    examples:
+    - name: List all properties of all interfaces on an IoT Plug and Play device.
+      text: >
+        az iot dt list-properties --login {iothub_cs} --source device
```

Comment:
For new users it helps describe every core element of a commands usage example, particularly new concepts such as `source`. I suggest explaining in the example what `-- source device` means. As well as `private`, `public` etc.

Reply:
updated help file